app/controllers/api/v1/plans_controller.rb

```diff
@@ -85,11 +85,16 @@ def input_params
             :id,
             :billable_metric_id,
             :charge_model,
-            { properties: {} },
-            group_properties: [
-              :group_id,
-              { values: {} },
-            ],
+            :instant,
+            {
+              properties: {},
+            },
+            {
+              group_properties: [
+                :group_id,
+                { values: {} },
+              ]
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.